services/director/src/simcore_service_director/producer.py

```diff
@@ -571,7 +571,7 @@ def _get_service_entrypoint(service_boot_parameters_labels: Dict) -> str:
         return node_details
 
 
-async def _get_node_details(app: web.Application, client: aiodocker.docker.Docker, service: Dict):
+async def _get_node_details(app: web.Application, client: aiodocker.docker.Docker, service: Dict) -> Dict:
```

Comment:
Are you using a tool to create all the Typings?

Reply:
I did use [mypy](https://mypy.readthedocs.io/en/stable/introduction.html)